_includes/payment-order-other-features.md

```diff
@@ -0,0 +1,1302 @@
+        "orderItems" : { "id": "/psp/paymentorders/{{ page.payment_order_id }}/orderItems" },
+        "metadata": { "id": "/psp/paymentorders/{{ page.payment_order_id }}/metadata" },
+        "payments": { "id": "/psp/paymentorders/{{ page.payment_order_id }}/payments" },
+        "currentPayment": { "id": "/psp/paymentorders/{{ page.payment_order_id }}/currentpayment" }
+    },
+    "operations": [
+        {
+            "method": "PATCH",
+            "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}",
+            "rel": "update-paymentorder-abort",
+            "contentType": "application/json"
+        },
+        {
+            "method": "GET",
+            "href": "{{ page.front_end_url }}/paymentmenu/{{ page.payment_token }}",
+            "rel": "redirect-paymentorder",
+            "contentType": "text/html"
+        },
+        {
+            "method": "GET",
+            "href": "{{ page.front_end_url }}/paymentmenu/core/scripts/client/px.paymentmenu.client.js?token={{ page.payment_token }}&culture=nb-NO",
+            "rel": "view-paymentorder",
+            "contentType": "application/javascript"
+        }
+    ]
+}
+```
+
+The response given when aborting a payment order is equivalent to a `GET`
+request towards the `paymentorders` resource, as displayed above.
+with its `state` set to `Aborted`.
+
+### Cancel
+
+{% include payment-order-cancel.md %}
+
+#### Reversal
+
+{% include payment-order-reversal.md %}
+
+### Transactions
+
+{% include transactions.md api_resource="paymentorders"
+documentation_section="checkout" %}
+
+#### Transaction
+
+{% include transaction.md api_resource="paymentorders"
+documentation_section="checkout" %}
+
+## Recurring Payments
+
+If you want to enable subsequent recurring – server-to-server – payments, you
+need to create a **recurrence token**. This token will be utilized after the
+initial payment order. **Recurring payments must be activated on the contract
+with Swedbank Pay in order to work.**
+
+### Recurrence Token
+
+* When initiating a `Purchase` payment order, you need to make sure that the
+  field `generateRecurrenceToken` is set to `true`. This recurrence token
+  will stored in the [authorization transaction][transaction]
+  sub-resource on the underlying payment resource.
+*  When initiating a `Verify` payment order, a recurrence token will be generated
+  automatically. This recurrence token is stored in the
+  [verification][verification-transaction]  sub-resource on the underlying
+  payment resource.
+
+You can view the current payment resource, containg the recurrence token and
+other payment instrument properties, by [expanding the sub-resource][expanding]
+[`currentpayment`][current-payment] when doing a `GET` request on the
+`paymentorders` resource.
+
+{:.code-header}
+**Request**
+
+```http
+GET /psp/paymentorders/{{ page.payment_order_id }}?$expand=currentpayment HTTP/1.1
+Host: {{ page.api_host }}
+```
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-tabs] Use spaces instead of tabs](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)